src/train-classifier/src/test/scala/VerifyTrainClassifier.scala

```diff
@@ -121,22 +130,34 @@ class VerifyTrainClassifier extends EstimatorFuzzingTest {
     TrainClassifierTestUtilities.trainScoreDataset(mockLabelColumn, catDataset, randomForestClassifier)
   }
 
-  test("Verify a trained classifier model can be saved and loaded") {
+  test("Verify saving and loading a trained classifier model") {
     val dataset: DataFrame = createMockDataset
 
+    val subfolder = "subdir"
+    val fileSystem = getFileSystem
+    fileSystem.setWorkingDirectory(new Path(fileSystem.getWorkingDirectory().toString(), subfolder))
+    val stagingInit = fileSystem.getWorkingDirectory().toString()
+    println("Running Spark job in " + stagingInit)
+
     val logisticRegressor = TrainClassifierTestUtilities.createLogisticRegressor(mockLabelColumn)
 
     val model = logisticRegressor.fit(dataset)
 
     val myModelName = "testModel"
-    lazy val dir = new File(myModelName)
+    lazy val dir = new File(subfolder + "/" + myModelName)
     try {
       model.write.overwrite().save(myModelName)
       // write a second time with overwrite flag, verify still works
       model.write.overwrite().save(myModelName)
       // assert directory exists
       assert(dir.exists())
 
+      val fileSystem = getFileSystem
```

Comment:
Awesome thanks for the quick turnaround!

Reply:
Thanks!